app/routes/Event.tsx

```diff
@@ -0,0 +1,12 @@
+import Calendar from '~/components/Calendar.client';
+
+// This will changed - TODO
+const Fallback = () => {
+  return <p>Loading IDE...</p>;
+};
+
+function Event() {
+  return typeof document !== 'undefined' ? <Calendar /> : <Fallback />;
```

Comment:
NIT: can get rid of the `return` keyword

Reply:
Removed return keyword